src/ImageSharp/PixelFormats/PixelImplementations/Bgr24.cs

```diff
@@ -185,6 +185,15 @@ public void FromRgb24(Rgb24 source)
             this.B = source.B;
         }
 
+        /// <inheritdoc/>
+        [MethodImpl(InliningOptions.ShortMethod)]
+        public void FromAbgr32(Abgr32 source)
+        {
+            this.R = source.R;
+            this.G = source.G;
+            this.B = source.B;
```

Comment:
Not as readable, but [less machinecode](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgBEBLbAcwDsJcGDmFwsAwhAAmMAIK9sAGwCeuDrgDcNek1YA5AK74YUYRq2NmLAEr7eQo+Ij4ADhwXGAysYBuwmGdoLVhs7DgcASTtjCGcvKF8wf01qGgBtDwwofTAMABlsJQh9DAAKfMLigGkOXkkWLwBHfRhQxQBKAF0tAGYGQSychhlgbihu0hoAbxoGWaZe4CUMGCHkufmGReWGACE1ueIFpZWAcX3Zw83jhitkgF8aNIyBvIKi0vL36tr6mCaWoTtLq0Xr9bIYXajUgoKYzA5HbZ7OEXBGnc4bLYrW7Ihg4y7EFAMABiUCcw1G4xK5LGpD6RSgiTaOOm1HW6wwAAs1NYGABeOn6Bkwazo9lc0QnPkCoUsM44sXcnZS3D0xIsJGsuYPTWzPG9AnE0n4anjAD6MipIxp0sZzPlbNmsAAZldtirBYkrDAXfznQwAKq8XDYJ3CmS4AA8JtIaFdMAAfCU/e6hW1RQ6GH7MTaYF6nSSnErfd6A0GQ2HJJIdscAPJOp24GClZOq3Pe2PkNP2h2ctRSwPB0MscMRzGxnZQlCJlsetv5o07Ls6hjau5AA)
```suggestion
            ref byte sourceRef = ref Unsafe.As<Abgr32, byte>(ref source);
            ref byte sourceRefFromB = ref Unsafe.AddByteOffset(ref sourceRef, 1);
            this = Unsafe.As<byte, Bgr24>(ref sourceRefFromB);
```
I'm not sure if I would take this suggestion (readability of code wise).

Reply:
I do take this suggestion, but adding a small code comment to explain the _magic_ happening here.